frontend/src/views/HomeView.vue

```diff
@@ -57,17 +62,92 @@
+        <v-row class="mx-md-10">
+          <v-col cols="6" class="text-center">{{ fundingAgreement.fundingAgreementNumber }}</v-col>
+          <v-col cols="6" class="text-center">{{ fundingAgreement.facilityName }}</v-col>
+        </v-row>
+      </v-card>
+      <br />
+      <h2 class="confirm-dialog-text d-flex flex-column text-center">You have one or more Funding Agreements ready to sign.</h2>
+      <div class="confirm-dialog-text d-flex flex-column text-center">Please visit the notification center for more details.</div>
+      <div class="confirm-dialog-text d-flex flex-column text-center">Would you like to take action now?</div>
+      <br />
+    </template>
+    <template #button>
+      <v-row justify="space-around">
+        <v-col cols="12" md="6" class="d-flex justify-center">
+          <AppButton id="dialog-go-back" :primary="false" size="large" width="200px" :loading="isLoading" @click="closeDialog">Not Now</AppButton>
+        </v-col>
+        <v-col cols="12" md="6" class="d-flex justify-center">
+          <AppButton id="dialog-cancel-application" size="large" min-width="250px" max-width="500px" :loading="isLoading" @click="goToFunding">Yes, take me to Funding</AppButton>
+        </v-col>
+      </v-row>
+    </template>
+  </AppDialog>
 </template>
 
 <script>
+import { mapState } from 'pinia'
+import { useAuthStore } from '@/stores/auth'
+import { FUNDING_AGREEMENT_STATUS_CODES } from '@/utils/constants'
 import OrganizationHeader from '@/components/organizations/OrganizationHeader.vue'
+import AppButton from '@/components/ui/AppButton.vue'
 import AppHeroImage from '@/components/ui/AppHeroImage.vue'
+import AppDialog from '@/components/ui/AppDialog.vue'
+import FundingAgreementService from '@/services/fundingAgreementService'
 import permissionsMixin from '@/mixins/permissionsMixin.js'
 
 export default {
   name: 'HomeView',
-  components: { AppHeroImage, OrganizationHeader },
+  components: { AppHeroImage, AppButton, AppDialog, OrganizationHeader },
   mixins: [permissionsMixin],
+  data() {
+    return {
+      isLoading: false,
+      isDisplayed: false,
+      fundingAgreements: [],
+    }
+  },
+  computed: {
+    ...mapState(useAuthStore, ['userInfo']),
+  },
+  async created() {
+    this.MAX_FACILITIES_DISPLAYED = 3
+    await this.loadFundingAgreements()
+  },
+  methods: {
+    closeDialog() {
+      this.isDisplayed = false
+    },
+    goToFunding() {
+      this.$router.push({ name: 'funding-overview' })
+    },
+    async loadFundingAgreements() {
+      this.isLoading = true
+
+      //I use for of here so I can break out of the loop when I hit 3 facilities.
+      //We show 3 max so the popup doesn't take over the screen, and we save on calls to Dynamics.
+      for (const fac of this.userInfo.facilities) {
+        const fa = await FundingAgreementService.getActiveFundingAgreementByFacilityId(fac.facilityId)
```

Comment:
I think we only need to get the list of Signature Pending FA? In that case, we can create a new service function in FAservice to get FA having a specific status (e.g.: getActiveFundingAgreementByFacilityIdAndStatus) and include statusCode = Signature Pending in the request filter.

Reply:
done